RELEASE/scripts/autoscend/auto_adventure.ash

```diff
@@ -117,6 +117,10 @@ boolean autoAdvBypass(int urlGetFlags, string[int] url, location loc, string opt
 	if (contains_text(page, combatPage)) {
 		auto_log_info("autoAdvBypass has encountered a combat! (param: '" + option + "')", "green");
 		run_combat(option);
+	} else if (contains_text(page, "Rainy Fax Dreams")) {
```

Comment:
on the edge of my knowledge here. Why is it better to add the check here instead of adding choice 970 to auto_choice_adv.ash?

Reply:
oh right i'll do that. i'm just a bit worried about this choice adventure screwing up because if you don't leave it correctly you can lose the rain

Reply:
yeah auto_choice_adv doesn't work for this one for some reason

Reply:
okay yeah that worked